config.fnl

```diff
@@ -0,0 +1,455 @@
+(local windows (require :windows))
+(local {:concat concat
+        :logf logf} (require :lib.functional))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Default Config
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; - It is not recommended to edit this file.
+;; - Changes may conflict with upstream updates.
+;; - Create a ~/.hammerspoon/private/config.fnl file instead.
```

Comment:
Would be nice to have an example of private/config.fnl in the docs and the link to the said example here

Reply:
The default config.fnl is already an example of a private/config.fnl. Users only need to copy config.fnl to either ~/.hammerspoon/private or ~/.spacehammer depending on if we merge #35.

Reply:
Had a better idea: When spacehammer starts we check for ~/.spacehammer/config and if it doesn't it copies the default to the custom dir. This way we can keep our documentation simpler by always directing users to their custom config. The changes are pushed to #35